app/javascript/utils/index.js

```diff
@@ -0,0 +1,28 @@
+const MS_IN_DAY = 24 * 60 * 60 * 1000;
+
+export const postedAtString = (createdAtString) => {
```

Comment:
This is the method that defines the logic for converting a date to "x days ago" that's displayed in the card. Check out the test for the next file for an explanation of the logic.